ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -1073,8 +1072,13 @@ public void shouldDropConnector() throws Exception {
     client.dropConnector(TEST_CONNECTOR).get();
 
     // Then:
-    final List<ConnectorInfo> connectors = client.listConnectors().get();
-    assertThat(connectors.size(), is(0));
+    assertThatEventually(() -> {
+      try {
+        return client.listConnectors().get().size();
+      } catch (InterruptedException | ExecutionException e) {
```

Comment:
Is this exception expected (under normal cases)? What would cause this?

Reply:
It's not. It's just that since this is a part of a lambda, we need to do a try-catch here in order for the code to compile.